### DISCUSSION TITLE
User blocking implementation

### DISCUSSION THREAD

**Discussion by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#619] [General]
I really do not like how blocking was implemented there was a lot more discussion on the old platform's GitHub about how it would be implemented blocking in Resonite feels like it was shoehorned in there was a lot more nuance and complexity to the blocking system that was just not implemented into the one we have now

I'm hoping that the blocking system is still open for recommendations and changes over time

here's a link to the original discussion
https://github.com/Neos-Metaverse/NeosPublic/issues/557

**Comment by ShrikeAlvaron** [comment]
What exactly are your issues with it?  If you have concerns about it you should outline what they are and what features you'd like to see added.

**Reply by epicEaston197** [reply]
In the issue I mentioned there was discussion on having temporary blocking and having a world host determine if blocking is enabled or not that can really help out with game worlds

I would highly recommend to read the issue I linked as it has really good discussion on the topic

**Comment by Frooxius** [comment]
As per our guidelines here: https://github.com/Yellow-Dog-Man/Resonite-Issues we are unable to use materials from past projects, so unfortunately we have to close this.

You are free to open a new issue with fresh feedback about the blocking feature in Resonite.
